### PR TITLE
Tabby Cat (combined) (#1118)

### PR DESCRIPTION
**Pull request by ljmwaugh**
prepare tabby_cat for PR

prepare tabby_cat for PR

done to fix federation_square from showing up here

* Merge branch 'instructlab:main' into main

* resolve issues with Tabby_Cat qna.yaml

make changes as specified by BJ

* Apply suggestions from code review

* submit cleaned up federation_square knowledge files

* Merge branch 'instructlab:main' into main

* Revert "submit cleaned up federation_square knowledge files"

This reverts commit dfd2ee97b1ffb7ec86ee200423711b4363a859c8.

If your PR is related to a contribution to the taxonomy, please, fill
out the following questionnaire. If not, replace this whole text and the
following questionnaire with whatever information is applicable to your PR.

**Describe the contribution to the taxonomy**

- ...
- ...
- ...

**Input given at the prompt**

```
   ...
```

**Response from the original model**

```
  ...
```

**Response from the fine-tuned model**

```
  ...
```

**Contribution checklist**

- [ ] The contribution was tested with `ilab generate`
- [ ] No errors or warnings were produced by `ilab generate`
- [ ] All [commits are signed off](https://github.com/instructlab/taxonomy/blob/main/CONTRIBUTING.md#legal) (DCO)
- [ ] The `qna.yaml` file contains at least 5 `seed_examples`
- [ ] The `qna.yaml` file was [linted](https://yamllint.com) and [prettified](https://onlineyamltools.com/prettify-yaml) ([yaml-validator](https://jsonformatter.org/yaml-validator) can do both)
- [ ] An `attribution.txt` file in the same folder as the `qna.yaml` file
- [ ] Content does not include PII or otherwise sensitive or confidential information
- [ ] Content does not include anything documented in the project's [Avoid these Topics](https://github.com/instructlab/taxonomy/blob/main/docs/SKILLS_GUIDE.md#avoid-these-topics) guidelines

### CODE REVIEW

**Comment by jjasghar** [comment]
@instructlab-bot precheck

File: knowledge/animals/tabby_cat/qna.yaml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+ - answer: |
+      The English term tabby originally referred to "striped silk taffeta",
+      from the French word tabis, meaning "a rich watered silk".
+   question: What is the origin of the name tabby?
+ - answer: |
+      The four known distinct patterns, each having a sound genetic
+      explanation, are the mackerel, classic, ticked, and spotted tabby patterns
+   question: What are the patterns that a tabby cat may have?
+ - answer: Personality and aggression vary widely from cat to cat, and is multifactorial.
+   question: Do tabby cats have a specific personalities?
```
Comment (kattekrab) [inline knowledge/animals/tabby_cat/qna.yaml:25]: > question: Do tabby cats have a specific personalities?
```   question: Do tabby cats have specific personalities?
```

Comment (ljmwaugh) [reply knowledge/animals/tabby_cat/qna.yaml]: I made the requested change and updated the PR

**Comment by jjasghar** [comment]
@instructlab-bot precheck

**Comment by jjasghar** [comment]
Hi! :wave:
With the new update and release of 0.18.0, our [base knowledge submissions](https://github.com/instructlab/taxonomy/?tab=readme-ov-file#knowledge-yaml-examples) have drastically changed. This has given us an opportunity to significantly improve the accuracy of our fine-tuned models, which is a major win for our community!

But alas, that has caused some changes that this PR isn't able to continue forward. We greatly appreciate the work you've put into this, and wish we could accept it but the amount of work to get it to be `version: 3` is unrealistic for us to ask.
If you _do_ want to do this, please don't hesitate to take the `qna.yaml` update it, and build it out. Otherwise, we hope you come back to our community with a new submission and help build out our Open Source community-driven LLM.

If you have any questions directly, please don't hesitate to comment on the PR or email me at [awesomeATinstructlab.ai](http://awesomeatinstructlab.ai/) (yes, really), and I'll do my best to answer your questions.

**Pull request closed without merging**